src/github.com/getlantern/autoupdate-server/server/asset_test.go

```diff
@@ -0,0 +1,16 @@
+package server
+
+import (
+	"fmt"
+	"testing"
+)
+
+const (
+	testAssetURL = `https://github.com/getlantern/autoupdate-server/releases/download/0.4.0/autoupdate-binary-darwin-amd64.v4`
+)
+
+func TestDownloadAsset(t *testing.T) {
+	if _, err := downloadAsset(testAssetURL); err != nil {
+		t.Fatal(fmt.Errorf("Failed to download asset: %q", err))
+	}
+}
```

Comment:
Oh, that makes sense! Maybe we could make that change when we switch our build scripts to use a single Makefile. I think just testing everything under getlantern would work well.